db/migrate/20180906155513_add_change_notes_to_documents.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddChangeNotesToDocuments < ActiveRecord::Migration[5.2]
+  def change
+    change_table :documents, bulk: true do |t|
```

Comment:
ooh I've not seen this `bulk` argument before.

Reply:
Me neither! Rubocop forced me to do this instead of 2 `add_columns`.

Reply:
He'll make us all redundant.